OrbitGl/FunctionsDataView.h

```diff
@@ -8,14 +8,17 @@
 #include "DataView.h"
 #include "capture_data.pb.h"
 
+class OrbitApp;
+
 class FunctionsDataView : public DataView {
  public:
-  FunctionsDataView();
+  FunctionsDataView(OrbitApp* app);
```

Comment:
Maybe this should be `explicit` now?

Reply:
Yesss. Thanks a lot. I missed that in quite a few locations - it seems! Fixing that now.

Reply:
Done. Thanks again!